wallet/how-to/use-non-evm-networks/starknet/create-a-simple-starknet-dapp.md

```diff
@@ -105,23 +128,38 @@ This object can control the connection process, including:
 You can configure these options as follows:
 
 ```typescript title="App.tsx"
-handleConnect({ modalMode: "alwaysAsk", modalTheme: "dark" });
+// Default connection.
+handleConnect()
+
+// Always show modal.
+handleConnect({ modalMode: "alwaysAsk" })
+
+// Never show modal.
+handleConnect({ modalMode: "neverAsk" })
+
+// Dark theme modal.
+handleConnect({
+  modalMode: "alwaysAsk",
+  modalTheme: "dark",
+})
+
+// Light theme modal.
+handleConnect({
+  modalMode: "alwaysAsk",
+  modalTheme: "light",
+})
 ```
```

Comment:
To better match the full code sample, I suggest renaming this step to something like "Display connection options," and updating the code sample to show the `return` statement with the HTML (minus the wallet information, which will be added in the next step).

Show the connection and disconnection options in the HTML button context like in the full example.